### PR TITLE
Move installation information out to a new page

### PR DESCRIPTION
**Pull request by roji**
Because our very first index page shouldn't contain stuff like GAC installation and App.Config...

### CODE REVIEW

File: conceptual/Npgsql/toc.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## [3.2](release-notes/3.2.md)
 ## [3.1](release-notes/3.1.md)
 ## [3.0](release-notes/3.0.md)
-# [FAQ](faq.md)
+# [Installation](installation.md)
 # [Compatibility Notes](compatibility.md)
```
Comment (YohDeadfall) [inline conceptual/Npgsql/toc.md]: Makes sense to swap installation and compatibility notes links to move the last one to release notes.

File: conceptual/Npgsql/toc.md [hunk 2]
```diff
@@ -28,6 +28,7 @@
 # [Bulk Copy](copy.md)
 # [Logging](logging.md)
 # [Visual Studio Integration](ddex.md)
+# [FAQ](faq.md)
 # [API Reference](../../obj/api/Npgsql/)
```
Comment (YohDeadfall) [inline conceptual/Npgsql/toc.md]: Should FAQ go after API reference?

File: conceptual/Npgsql/toc.md (hunk 1 above)
Comment (roji) [reply conceptual/Npgsql/toc.md]: I swapped them, but I'm not sure about moving compatibility notes inside the release notes... Seems like it should be visible on the top-level menu no? We can always change later.

File: conceptual/Npgsql/toc.md (hunk 2 above)
Comment (roji) [reply conceptual/Npgsql/toc.md]: I'm not sure, I don't think so... FAQ is still conceptual documentation (articles in English), API documentation is a really different kind of docs... We should review the FAQ stuff too at some point, I think it's not very updated.

**Pull request merged**